inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/InlongGroupServiceImpl.java

```diff
@@ -435,7 +435,7 @@ public InlongGroupInfo doDeleteCheck(String groupId, String operator) {
 
         // If the status allowed logic delete, all associated info will be logically deleted.
         // In other status, you need to delete the related "inlong_stream" first.
-        if (!GroupStatus.allowedLogicDelete(curState)) {
+        if (!GroupStatus.notAllowedDelete(curState)) {
             int count = streamService.selectCountByGroupId(groupId);
```

Comment:
Maybe, `!GroupStatus.notAllowedDelete(curState)` modify to `GroupStatus.notAllowedDelete(curState)`

Reply:
When the inlong group supports the delete operate, we should check whether there are undeleted streams.